tests/ballerina-examples-test/build.gradle

```diff
@@ -45,98 +45,382 @@ task copyDistribution() {
+            'http-timeout',
+            'http-retry',
+            'http-failover',
+            'http-load-balancer',
+            'http-circuit-breaker',
+            'content-based-routing',
+            'header-based-routing',
+            'http-filters',
+            'http-compression',
+            'http-100-continue',
+            'http-access-logs',
+            'http-cors',
+            'request-with-multiparts',
+            'mutual-ssl',
+            'basic-https-listener-client',
+            'https-listener',
+            'http-disable-chunking',
+            'base-path-and-path',
+            'hello-world-service',
+            'tuple-match-statement'
+    ]
+    def mockList = [
+            'strings',
+            'math-functions',
+            'cache',
+            'user-defined-error',
+            'trap',
+            'check',
+            'error-handling',
+            'error-destructure-binding-pattern',
+            'record-destructure-binding-pattern',
+            'tuple-destructure-binding-pattern',
+            'error-typed-binding-pattern',
+            'record-typed-binding-pattern',
+            'tuple-typed-binding-pattern',
+            'json-to-xml-conversion',
+            'json-record-map-conversion',
+            'json-access',
+            'json-arrays',
+            'json-objects',
+            'json',
+            'immutable-values',
+            'clone',
+            'optional-field-access',
+            'length',
+            'equality',
+            'compound-assignment-operators',
+            'object-type-reference',
+            'anonymous-objects',
+            'abstract-objects',
+            'object-assignability',
+            'object-methods',
+            'objects',
+            'record-type-reference',
+            'record-optional-fields',
+            'anonymous-records',
+            'records',
+            'string-template',
+            'range-expressions',
+            'constants',
+            'any-type',
+            'anydata-type',
+            'optional-type',
+            'byte-type',
+            'union-type',
+            'table',
+            'maps',
+            'tuple-type',
+            'arrays',
+            'values',
+            'elvis-operator',
+            'type-guard',
+            'type-test-expression',
+            'error-match-statement',
+            'record-match-statement',
+            'match',
+            'foreach',
+            'while',
+            'if-else',
+            'functional-iteration',
+            'closures',
+            'anonymous-functions',
+            'function-pointers',
+            'quoted-identifiers',
+            'functions-with-rest-parameter',
+            'functions-with-defaultable-parameters',
+            'functions-with-required-parameters',
+            'functions',
+            'var',
+            'variables',
+            'modules',
+            'hello-world-client',
+            'hello-world'
+    ]
+    def stdEMock = [
```

Comment:
@ramithaYO Shall we add comments for these lists and mention what will be done with these lists?